test/db/tools/rz_bin

```diff
@@ -155,87 +155,87 @@ RUN
 
 NAME=rz-bin -SSS
 FILE=bins/elf/analysis/x86-helloworld-gcc
-CMDS=!rz-bin -SSS ${RZ_FILE}|wc -l|awk "{print \$1}"
+CMDS=!!rz-bin -SSS ${RZ_FILE}|wc -l|awk "{print \$1}"
 EXPECT=<<EOF
 13
 EOF
 RUN
 
 NAME=rz-bin -z pe
 FILE=bins/pe/a.exe
-CMDS=!rz-bin -z ${RZ_FILE}|wc -l|awk "{print \$1}"
+CMDS=!!rz-bin -z ${RZ_FILE}|wc -l|awk "{print \$1}"
 EXPECT=<<EOF
 7
 EOF
 RUN
 
 NAME=rz-bin -zz pe
 FILE=bins/pe/ioli/w32/crackme0x00.exe
-CMDS=!rz-bin -zz ${RZ_FILE} | grep "Password:"
+CMDS=!!rz-bin -zz ${RZ_FILE} | grep "Password:"
 EXPECT=<<EOF
 86  0x00002619 0x00404019 10  11   .rdata  ascii   Password: 
 EOF
 RUN
 
 NAME=rz-bin -zzz pe
 FILE=bins/pe/ioli/w32/crackme0x00.exe
-CMDS=!rz-bin -zzz ${RZ_FILE} | grep "Password:"
+CMDS=!!rz-bin -zzz ${RZ_FILE} | grep "Password:"
```

Comment:
I guess this kind of change is now required because the `|` is handled by the rz-shell, so you have to redirect the output/err to rzcons, so that it can be later sent to `grep`, right?

Reply:
Yes. because !! does fetch stdout, and ! does not